packages/expo-document-picker/src/ExpoDocumentPicker.web.ts

```diff
@@ -20,7 +20,7 @@ export default {
     const input = document.createElement('input');
     input.style.display = 'none';
     input.setAttribute('type', 'file');
-    input.setAttribute('accept', type);
+    input.setAttribute('accept', type as string);
```

Comment:
Here we've got a problem 🤔 
This is the available typing: `type: string | string[]` and you narrow it down to `string` without any check. What about `string[]`? Is it impossible to pass `string[]` on the web? 🤔 If so we need to document this and properly handle `string[]` here 🤔